src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5576,6 +5601,7 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						int v = atoi(val);
 						if (v >= 0) {
 							profile->rtp_timeout_sec = v;
+							sofia_set_media_flag(profile, SCMF_RTP_TIMEOUT_SEC);
```

Comment:
We should have a separate configuration field that set rtp-timeout-sec for other channel